src/main/java/WayofTime/bloodmagic/ritual/types/RitualEternalSoul.java

```diff
@@ -0,0 +1,127 @@
+package WayofTime.bloodmagic.ritual.types;
+
+import WayofTime.bloodmagic.BloodMagic;
+import WayofTime.bloodmagic.altar.IBloodAltar;
+import WayofTime.bloodmagic.block.BlockLifeEssence;
+import WayofTime.bloodmagic.ritual.*;
+import WayofTime.bloodmagic.util.helper.NetworkHelper;
+import WayofTime.bloodmagic.util.helper.PlayerHelper;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+@RitualRegister("eternal_soul")
+public class RitualEternalSoul extends Ritual {
+
+    private UUID owner;
```

Comment:
Why are these fields stored?

Reply:
because I'm using owner more than once in the code /shrug

Reply:
Yes but why is it stored as a field and not a local var?

Reply:
Paranoid of issues resulting from stuff not being instantiated resulting in irrational thinking because it's irrelevant for a local variable.